setup.py

```diff
@@ -135,7 +135,8 @@ def run(self):
             'six',
             'pytest-xdist',
             'virtualenv',
-        ]
+        ],
+        'toml': ['coverage[toml] >= 5'],
```

Comment:
Isn't the `>=5` part redundant? It's already specified more precisely in `install_requires`.

Reply:
Good point. I don't know.